### PR TITLE
[PORT] Port base_suspend_security to 9.0

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

File: base_suspend_security/__openerp__.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "test": [
     ],
     "auto_install": False,
-    'installable': False,
+    'installable': True,
```
Comment (JonathanNEMRY) [inline base_suspend_security/__openerp__.py]: I think version number should be upgraded too?

**Comment by gurneyalex** [comment]
@adrienpeiffer can you also bump the version number in **openerp**.py ?

**Comment by lmignon** [comment]
@adrienpeiffer @gurneyalex @hbrunn IMO We should advertise that this module must be used with caution in a multi companies environment.  Indeed, when you suspend the security you also suspend record rules and in some cases I've observed that the value of some fields is not linked to the current company.

**Comment by pedrobaeza** [comment]
And a workaround/solution is to include in the context the key `force_company` with the id of the company that must be treated.

**Comment by lmignon** [comment]
@pedrobaeza I'm not sure it's a solution since the record rules are not run at all...

**Comment by pedrobaeza** [comment]
Well, it's not a complete solution for example for many2many records (like the taxes in the product), but it works with property/company_dependent fields.

**Comment by hbrunn** [comment]
@lmignon how is `sudo()`'s behavior here? I think we should do what this function does for consistency

**Comment by lmignon** [comment]
@gurneyalex version bumped

**Comment by hbrunn** [comment]
:+1:

**Comment by sebalix** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**